modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro

```diff
@@ -194,6 +194,42 @@ definition {
 		PortletEntry.inputTitle(title = "${title}");
 	}
 
+	macro gotoManageTranslations {
+		Click(locator1 = "TranslationManager#MANAGE_TRANSLATIONS");
+
+		if (isSet(searchLanguage)) {
+			Type(
+				locator1 = "ManageTranslations#SEARCH_FIELD",
+				value1 = "${searchLanguage}");
+		}
+
+		if (isSet(addLanguage)) {
+			Click(locator1 = "Button#PLUS");
+
+			Click(
+				key_locale = "${addLanguage}",
+				locator1 = "ManageTranslations#DROPDOWN_MENU_ITEM_LANGUAGE");
+		}
+
+		if (isSet(deleteLanguage)) {
+			Click(
+				key_language = "${deleteLanguage}",
+				locator1 = "ManageTranslations#LIST_ITEM_DELETE_LANGUAGE_BUTTON");
+		}
+
+		if (isSet(close)) {
+			Click(locator1 = "ProcessBuilderKaleoDesignerReact#BUTTON_CLOSE_MODAL");
+		}
+
+		if (isSet(cancel)) {
+			Click(locator1 = "Button#CANCEL");
+		}
+
+		if (isSet(done)) {
+			Click(locator1 = "Button#DONE");
+		}
+	}
```

Comment:
Hey @Italo-gabriel , i can see why you had the idea of make a macro complete like this, but this macro it's not just going to the Manage Translations, it's doing a lot of other actions too, it would not be clear to other QAs/etc that this macro can be so powerfull, that why i believe it would be more assertive if we create some other macros with the respective actions.